Lib/selectors.py

```diff
@@ -357,7 +357,7 @@ def register(self, fileobj, events, data=None):
             poller_events |= self._EVENT_WRITE
         try:
             self._selector.register(key.fd, poller_events)
-        except Exception:
+        except:
```

Comment:
I'm not sure I understand. Exceptions on Python 3 are chained, so in case of error on `unregister` the previous exception will still be shown. As such, there's no need of the `try/finally`.